src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsJavaDocsTest.java

```diff
@@ -547,6 +548,7 @@ private static boolean isSetterMethod(DetailAST ast) {
         private static String getJavaDocText(DetailAST node) {
             final String text = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n<document>\n"
                     + node.getFirstChild().getText().replaceAll("(^|\\r?\\n)\\s*\\* ?", "\n")
+                            .replaceAll("@noinspection.*\\r?\\n", "")
```

Comment:
Should we ensure `@noinspection` starts the line of the javadoc?

Reply:
Yes we should. Done.